packages/backend-core/src/environment.ts

```diff
@@ -134,10 +134,10 @@ const environment = {
   BUDIBASE_ENVIRONMENT: process.env.BUDIBASE_ENVIRONMENT,
   JS_BCRYPT: process.env.JS_BCRYPT,
   JWT_SECRET: process.env.JWT_SECRET
-    ? createSecretKey(Buffer.from(process.env.JWT_SECRET))
+    ? createSecretKey(process.env.JWT_SECRET, "utf8")
```

Comment:
This is a strange case... We are using node 20, and based on their docs, it should accept "buffer" as a parameter. But the types don't allow it. Until now it was working as we were wrongly using node 22 types